cf-operator-deployment.yaml

```diff
@@ -0,0 +1,78 @@
+#
```

Comment:
The name of the file is too generic - people could think it's something else.
How about `dev-env-havener.yaml` ?

Reply:
sounds good.